packages/core/src/dispatcher.ts

```diff
@@ -367,9 +367,10 @@ export class Dispatcher {
       Metrics.count(COMMITS_STORED, 1)
       return cid
     } catch (e) {
-      this._logger.err(`Error while storing commit to IPFS: ${e}`)
+      const errorMessage = `Error in repository.ts.storeInitEvent: Error - ${e} while storing commit to ipfs: ${e.message} | recon.mode: ${this.recon.enabled}`
```

Comment:
I think this meant to say we are in dispatcher.ts, not repository.ts.

But to be honest I don't really love putting the file name into the error message.  If the message is descriptive and unique enough, it should be easy to grep for to find what file it is in.  Putting the file name into the message feels brittle in case the code is ever refactored, and also kind like leaking information that isn't relevant to our users.  I'd be okay with a stack trace though as that at least will update if the message moves and feels like a more standard thing devs know how to use.

Reply:
Also this doesn't really feel like the right place to log the recon mode either.  I think global configuration like that should be logged once at startup, rather than every time there is an error.